site/src/pages/news/20220708-newsletter-4.mdx

```diff
@@ -0,0 +1,156 @@
+---
+date: 2022-07-08
+---
+
+# Armeria Newsletter vol. 4
+
+<video class="hideOnReducedMotion" src="../../images/armeria.m4v" preload="none" autoplay muted loop style="width: 282px; height: 112px;">
+  <img src="../../images/armeria.gif" loading="lazy" width="282" height="112" />
+</video>
+
+## From the devs
+
+<div class="noBullet">
+
+- ✨ We released several releases since our last newsletter with a bunch of new exciting features!
+  - [1.13.4](https://armeria.dev/release-notes/1.13.4/)
+    - Security fix(CWE-22)
+  - [1.14.0](https://armeria.dev/release-notes/1.14.0/)
+    - Experimental support for Scala 3
+    - Fluent conversion of <type://HttpResponse> into a desired type using <type://WebClient>
+    - Various build options for gRPC using <type://GrpcClientBuilder>
+  - [1.14.1](https://armeria.dev/release-notes/1.14.1/)
+    - Additional logs for <type://EmptyEndpointGroupException>
+  - [1.15.0](https://armeria.dev/release-notes/1.15.0/)
+    - User-friendly gRPC health check protocol
+    - Multipart file uploads in annotated services
+    - Preliminary GraphQL support in <type://DocService>
+    - Refactored DMS resolver cache with higher hit ratio and reduced traffic
+    - New metrics that help you get alerts before your TLS certificates are expired
+  - [1.16.0](https://armeria.dev/release-notes/1.16.0/)
+    - New metrics for TLS handshake results
+    - Handling <type://Decorator> annotation with <type://GrpcService> for your gRPC stubs
+    - Programmatic <type://Flags> configuration via Java SPI
+  - [1.17.0](https://armeria.dev/release-notes/1.17.0/)
+    - Easier way to send and receive RESTful APIs
+    - Ability to limit the maximum length of Thrift messages
+    - Injecting dependencies in annotations
+    - Support for Kotlin nullable type(`?`) in annotated services
+
+</div>
+<div class="noBullet">
+
+- 👂 Armeria team is hosting [bi-weekly virtual office hours](https://armeria.dev/community/office-hours/).
+  - It was great talking with all of you through the video chat. And we're still listening. :)
```

Comment:
I think the sentence sounds a little vague. Did you mean we're still doing virtual hours?

```suggestion
  - It was great talking with all of you through video chat. We hope we can see you again in the near future. :)
```